bin/ci

```diff
@@ -69,6 +69,10 @@ prepare_system() {
   on_linux 'echo '"'"'{"ipv6":true, "fixed-cidr-v6":"2001:db8:1::/64"}'"'"' | sudo tee /etc/docker/daemon.json'
   on_linux sudo service docker restart
 
+  # TODO: Delete Me
+  on_linux cat /etc/passwd
+  on_linux cat /etc/group
```

Comment:
These need to use `with_build_env` actually. Place them at the start of `build()`.